http/http.cpp

```diff
@@ -35,15 +33,12 @@ void Http::connect(std::string url) {
 }
 
 std::string Http::makeRequest(const char *req) {
+    constexpr auto BUFFER_SIZE = 10000;
     send(Http::sck, req, std::string(req).length(), 0);
-    char* buffer = (char *)malloc(BUFFER_SIZE);
```

Comment:
if you really were to malloc, the most C++-ish thing here would've been
`auto buffer = static_cast<char*>(std::malloc(BUFFER_SIZE));`